resources/prefs.json

```diff
@@ -1,7 +1,7 @@
 {
   "dom.bluetooth.enabled": false,
   "dom.bluetooth.testing.enabled": false,
-  "dom.canvas-text.enabled": false,
+  "dom.canvas-text.enabled": true,
```

Comment:
Please revert this change. We do not want this to be exposed by default, since it is incomplete. Anybody that needs to run tests with this API present can enable it explicitly.